driverService/src/test/java/com/modsen/driverservice/util/TestData.java

```diff
@@ -0,0 +1,200 @@
+package com.modsen.driverservice.util;
+
+import com.modsen.driverservice.dto.CarRequestDto;
+import com.modsen.driverservice.dto.CarResponseDto;
+import com.modsen.driverservice.dto.DriverRequestDto;
+import com.modsen.driverservice.dto.DriverResponseDto;
+import com.modsen.driverservice.dto.UserRatingDto;
+import com.modsen.driverservice.model.Car;
+import com.modsen.driverservice.model.Driver;
+
+import java.util.List;
+
+public class TestData {
+
+    public static final String URL_CAR = "/api/v1/cars";
+    public static final String URL_CAR_ID = URL_CAR + "/{carId}";
+    public static final String URL_CAR_DRIVER_ID = URL_CAR + "/driver/{driverId}";
+
+    public static final String URL_DRIVER = "/api/v1/drivers";
+    public static final String URL_DRIVER_ID = URL_DRIVER + "/{driverId}";
+    public static final String OFFSET = "offset";
+    public static final String LIMIT = "limit";
+
+    public static final Integer OFFSET_VALUE = 0;
+    public static final Integer LIMIT_VALUE = 5;
+    public static final Integer INSUFFICIENT_OFFSET_VALUE = -1;
+    public static final Integer INSUFFICIENT_LIMIT_VALUE = -1;
+    public static final Integer EXCEEDED_OFFSET_VALUE = 100;
+    public static final Integer EXCEEDED_LIMIT_VALUE = 100;
+
+    public static final Long CAR_ID = 1L;
+    private static final String color = "red";
```

Comment:
Для именования констант следует использовать SCREAMING_SNAKE_CASE:
```suggestion
    private static final String COLOR = "red";
```